lib/plugins/aws/provider/awsProvider.test.js

```diff
@@ -845,6 +847,7 @@ describe('AwsProvider', () => {
     const AwsProviderProxyquired = proxyquire('./awsProvider.js', {
       'aws-sdk': awsStub,
     });
+    sinon.stub(userStats, 'track');
```

Comment:
This should be done in `before` (otherwise it may end being stubbed for other test files - Mocha first runs describe bodies of all tests to be run.. and then follows with running tests).

Same way stub should be cleared in `after` so it doesn't leak to other tests (in mocha all test files are run in same Node.js process)